bin/nic.pl

```diff
@@ -313,16 +313,23 @@ sub getHomeDir {
 }
 
 sub loadConfig {
-	open(my $cfh, "<", getHomeDir()."/.nicrc") or return;
-	while(<$cfh>) {
-		if(/^(.+?)\s*=\s*\"(.*)\"$/) {
-			my $key = $1;
-			my $value = $2;
-			$CONFIG{$key} = $value;
-		}
-	}
+    open(my $cfh, "<", getHomeDir()."/.nicrc") or return;
+    while(<$cfh>) {
+        # Remove leading and trailing whitespace
+        s/^\s+|\s+$//g;
+        if (/^(.+?)\s*=\s*"(.*)"$/) {
+            my $key = $1;
+            my $value = $2;
+            # Remove leading and trailing whitespace from key and value
+            $key =~ s/^\s+|\s+$//g;
```

Comment:
With the above change, this would no longer be necessary as the key is matched irrespective of whitespace and the equal sign is also cleared of whitespace before splitting the pair